lib/src/model/amount.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:decimal/decimal.dart';
+
+class Amount {
```

Comment:
I already added it in GetProfileHandler.

Reply:
i don't understand what you're trying to say here. this class didn't exist as-is, and i don't get why you're talking about getprofilehandler